src/containers/singleEvent/index.tsx

```diff
@@ -27,6 +29,43 @@ const CenteredContainer = styled.div`
   justify-content: center;
 `;
 
+const AdminActionButtonList = styled.div`
+  display: flex;
+`;
+
+const GreenButton = styled(Button)`
```

Comment:
I would prefer if we had a base "StyledButton" and then had Green, Red, and Gray button extend from that base button instead.